Core/GUI/GameTooltipHooks.lua

```diff
@@ -510,6 +510,26 @@ local function processItem(id)
 	end
 end
 
+--- Handles updating an item tooltip to add additional information about attempts
+---@param tooltip GameTooltip
+---@param tooltipData TooltipData
+local function onTooltipSetItem(tooltip, tooltipData)
+	if tooltip ~= _G.GameTooltip and tooltip ~= _G.ItemRefTooltip then
+		return
+	end
+
+	local _, itemLink = tooltip:GetItem()
+	if not itemLink or type(itemLink) ~= "string" then
```

Comment:
```suggestion
	if type(itemLink) ~= "string" then
```

Unless I'm missing something, the only possible failure case is `itemLink == nil`, but `type(nil) == "nil"`:

![image](https://github.com/user-attachments/assets/d815c1cb-694a-46da-a605-19b852ddc18e)